### PR TITLE
WorkOrders List | Documentation

### PR DESCRIPTION
**Pull request by tiagobastos**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3559,6 +3559,75 @@ Update the current timer. Only possible if there is a timer running.
...
+                    + sent
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + starts_at
+                + order (enum[string], optional)
+                    + Members
+                        + asc
```
Comment (luisaterrabrito) [inline apiary.apib:3577]: missing desc

**Review by luisaterrabrito — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]

File: src/09-workorders/work-orders.apib [hunk 2]
```diff
@@ -1,5 +1,74 @@
 ## Work Orders [/workOrders]
 
+### workOrders.list [GET /workOrders.list]
+
+Get a list of work orders.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + filter (object, optional)
```
Comment (dietrichm) [inline src/09-workorders/work-orders.apib]: B: filtering is not in scope of the first story. There's a separate story to be picked up for that later.

File: src/09-workorders/work-orders.apib [hunk 3]
```diff
@@ -1,5 +1,74 @@
...
+                        + contact
+                        + company
+                + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+            + state (enum[string], optional)
+                + Members
+                    + draft
+                    + finalized
+                    + sent
+        + page (Page, optional)
+        + sort (array, optional)
```
Comment (dietrichm) [inline src/09-workorders/work-orders.apib:11]: Not sure if sorting is in scope as well. We could still create a separate story or already include it in this one.

**Review by dietrichm — DISMISSED** [review]

File: src/09-workorders/work-orders.apib [hunk 4]
```diff
@@ -1,5 +1,75 @@
...
+                        + draft
+                        + finalized
+                        + sent
+                + customer (object)
+                    + type (enum[string])
+                        + Members
+                            + contact
+                            + company
+                    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string)
+                + starts_at: `2017-05-26T10:01:49+00:00` (string)
```
Comment (JordyDeGroeve) [inline src/09-workorders/work-orders.apib]: NTH: keep the start date a bit closer to the created and updated date

**Review by JordyDeGroeve — APPROVED** [review]

File: src/09-workorders/work-orders.apib (hunk 3 above)
Comment (tiagobastos) [reply src/09-workorders/work-orders.apib:11]: Pagination and sorting were added to the same user story.

File: src/09-workorders/work-orders.apib (hunk 2 above)
Comment (tiagobastos) [reply src/09-workorders/work-orders.apib]: Removed.

**Review by WouterSioen — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Review by luisaterrabrito — APPROVED** [review]

**Comment by JordyDeGroeve** [comment]
✅ was able to create the apiary correctly

File: apiary.apib [hunk 5]
```diff
@@ -3559,6 +3559,64 @@ Update the current timer. Only possible if there is a timer running.
...
+                        + Members
+                            + contact
+                            + company
+                    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string)
+                + starts_at: `2019-05-08T10:12:01+00:00` (string)
+                + description: `Pipework intervention` (string)
+                + department (object)
+                    + type: `department` (string)
+                    + id: `5e90eb0a-b502-4344-aa0f-3b8525af6186` (string)
+                + layout (object)
```
Comment (tiagobastos) [inline apiary.apib]: @dietrichm Are we going to return the layout here? 😅 
Copied this from the proposal. But seams weird doing this here and not in the `.info`

Comment (dietrichm) [reply apiary.apib]: Yes indeed. This needs to be removed.

File: apiary.apib [hunk 6]
```diff
@@ -3559,6 +3559,64 @@ Update the current timer. Only possible if there is a timer running.
...
+                        + finalized
+                        + sent
+                + customer (object)
+                    + type (enum[string])
+                        + Members
+                            + contact
+                            + company
+                    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string)
+                + starts_at: `2019-05-08T10:12:01+00:00` (string)
+                + description: `Pipework intervention` (string)
```
Comment (dietrichm) [inline apiary.apib]: B: needs to be nullable (see .info). Repeated for department, milestone, event, location and distance.

**Review by dietrichm — DISMISSED** [review]

**Review by WouterSioen — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Comment by JordyDeGroeve** [comment]
✅ Apiary generates and looks OK. 
~Just a general Q, how are the users able to see if this is nullable on an info response?~
Found it: by clicking the JSON scheme

**Review by dietrichm — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Pull request merged**